gwt/pom.xml

```diff
@@ -40,39 +40,10 @@
       <classifier>sources</classifier>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <classifier>sources</classifier>
```

Comment:
Aha. And they need both the .java and .class files for Dagger/JSR330?

In that case, should we be including the non-sources version of javax.inject? I don't see it in this file.